packages/rehype-mathjax/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Minimum TypeScript Version: 3.2
+import {Plugin} from 'unified'
+
+// Should be ported back to MathJax repo
+// http://docs.mathjax.org/en/latest/options/output/svg.html#the-configuration-block
+interface MathJaxSVGOptions {
```

Comment:
For our own typings, could you do camelcase like `MathJaxSvgOptions` instead of ~`MathJaxSVGOptions`~?